index.html

```diff
@@ -215,7 +218,50 @@ <h5 class="mb">Would you like to know more information on bentos around you?</h5
 		        'opacity': 1 - (scrollPos / 250)
 		    });
 		}
+
+		// Check if element is on screen or not
+
+		$.fn.isOnScreen = function() {
+			var viewport = {};
+		    viewport.top = $(window).scrollTop();
+		    viewport.bottom = viewport.top + $(window).height();
+		    var bounds = {};
+		    bounds.top = this.offset().top;
+		    bounds.bottom = bounds.top + this.outerHeight();
+		    return ((bounds.top <= viewport.bottom) && (bounds.bottom >= viewport.top));
+		};
+		
+		Math.easeInQuad = function (t, b, c, d) {
+	t /= d;
+	return c*t*t + b;
+};
+
+		$.fn.countUpAnimate = function(start, target) {
+			var elements	= this;
+			var clr 		= null;
+			var count 		= start;
+			var fixed_step 	= 30;
+			var count_step	= Math.round((target - start) / fixed_step);
+
+			var loop;
+			(loop = function () {
```

Comment:
hmm, as this function is a IIFE (immediately invoked function execution), i think we can do without the variable declarations for `loop`? we would still need `inloop` var declared since you are passing it to the `setTimeout` function...

```javascript
(function (clr) {
   var inloop;
   clearTimeout(clr);
   (inloop = function () {
      if (count >= target) return;
      count = count + count_step > target ? target : count + count_step;
      elements.text(count);
      clr = setTimeout(inloop, 32);
   })();
})(null);
```